kubevirt/k6t-only-instancetype/k6t-require-instancetype.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: k6t-require-instancetype
+  annotations:
+    policies.kyverno.io/title: Require instanceType
+    policies.kyverno.io/category: KubeVirt
+    kyverno.io/kyverno-version: 1.6.2
+    policies.kyverno.io/minversion: 1.6.2
+    kyverno.io/kubernetes-version: "1.21-1.25"
+    policies.kyverno.io/subject: VirtualMachine
+    policies.kyverno.io/description: >-
+      Check VirtualMachines and validate that they are not using a domain template (`.spec.template.domain`)
+      but instead are based on instaceTypes and preferences only.
+spec:
+  validationFailureAction: enforce
+  rules:
+  - name: k6t-dont-allow-domain-template
+    match:
+      any: 
+      - resources:
+          kinds:
+          - VirtualMachine
+    validate:
+      message: "VirtualMachines must only use instanceTypes and preferences, a domain template is not allowed."
```

Comment:
Based on your message, it looks like your pattern may be wrong. This pattern effectively means, "make sure a VirtualMachine has `spec.template.domain` with something inside it.

Reply:
Thanks! I also noticed that I'm checking the wrong path. I'll fix both.